sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -859,7 +911,13 @@ private long getEstimatedSizeBytesBasedOnSamples(List<SampleRowKeysResponse> sam
 
     @Override
     public void validate() {
-      checkArgument(!tableId.isEmpty(), "tableId cannot be empty");
+        if (tableId == null) {
+            throw new IllegalArgumentException("tableId not specified");
+        } else if (!tableId.isAccessible()) {
+            LOG.warn("Skipping validation for tableId");
```

Comment:
explain why -- use something like what I suggested above (not available at this point because its a parameter, etc.)